Project/Engine/src/externals/imgui/imgui_impl_dx12.cpp

```
@@ -85,13 +85,90 @@ struct ImGui_ImplDX12_RenderBuffers
+    // Render buffers
+    UINT                            FrameIndex;
+    ImGui_ImplDX12_RenderBuffers*   FrameRenderBuffers;
+
+    ImGui_ImplDX12_ViewportData(UINT num_frames_in_flight)
+    {
+        CommandQueue = nullptr;
+        CommandList = nullptr;
+        RtvDescHeap = nullptr;
+        SwapChain = nullptr;
+        Fence = nullptr;
+        FenceSignaledValue = 0;
```

Cppcheck (2.10-2)

[performance] Variable 'FenceSignaledValue' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)